rollpe/user/apis.py

```diff
@@ -28,32 +40,109 @@ def logout_api(request):
     refresh_token = request.data.get("refresh")
 
     if not refresh_token:
-        return Response(data={"error": "Refresh token이 없습니다."}, status=400)
+        return Response(data={"message": "Refresh token이 없습니다."}, status=400)
```

Comment:
```suggestion
        return Response(msg= "Refresh token이 없습니다.", status=400)
```

이런식으로 작성하면 
response 안에 들어갈 수 있도록 Response 변경하겠습니다 !!!